### PR TITLE
testing the svg generation

### PR DESCRIPTION
**Pull request by JustinMacaulay**
a component to test and lay the framework for the document generation in with SVG's

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -40,22 +40,22 @@ services:
...
-          REACT_APP_KEYCLOAK_REALM: "${KEYCLOAK_REALM}"
-          REACT_APP_KEYCLOAK_CLIENT_ID: "${KEYCLOAK_CLIENT_ID}"
-          REACT_APP_API_BASE_URL: "${REACT_APP_API_BASE_URL}"
-      depends_on:
-        - roadside-forms-api
-      networks:
-        - docker-network
-      ports:
-        - "8080:5000"
+    # web_app:
```
Comment (NavpreetGrewal) [inline docker-compose.yml:43]: is this change needed?

Comment (JustinMacaulay) [reply docker-compose.yml:43]: I just commented it out cause I only use it when testing specific functionality. I run NPM to host the app when I am developing for hot reload.

Comment (NavpreetGrewal) [reply docker-compose.yml:43]: Personally, I also use NPM for frontend but I think we should not commit it as commented out code as it will be good for someone new to the app to run it with docker as a complete app instead of trying to figure out how to run frontend separately. my two cents.

**Review by NavpreetGrewal — APPROVED** [review]
